Wurstpack/wurstscript/lib/systems/Knockback3.wurst

```diff
@@ -0,0 +1,217 @@
+package Knockback3
+
+import Terrain
+import MapBounds
+import LinkedListModule
+import HashMap
+
```

Comment:
What do you mean "put the example here". The example is a hotdoc comment next to `Knockback3.add` class method. I can put it just below `public class Knockback3` ?